manim/_config/utils.py

```diff
@@ -1027,22 +1030,28 @@ def dry_run(self, val: bool) -> None:
             )
 
     @property
-    def use_js_renderer(self):
+    def use_webgl_renderer(self):
         """Whether to use JS renderer or not (default)."""
-        return self._d["use_js_renderer"]
+        return self._d["use_webgl_renderer"]
 
-    @use_js_renderer.setter
-    def use_js_renderer(self, val: bool) -> None:
-        self._d["use_js_renderer"] = val
+    @use_webgl_renderer.setter
+    def use_webgl_renderer(self, val: bool) -> None:
+        self._d["use_webgl_renderer"] = val
         if val:
             self["disable_caching"] = True
 
-    js_renderer_path = property(
-        lambda self: self._d["js_renderer_path"],
-        lambda self, val: self._d.__setitem__("js_renderer_path", val),
+    webgl_renderer_path = property(
+        lambda self: self._d["webgl_renderer_path"],
+        lambda self, val: self._d.__setitem__("webgl_renderer_path", val),
         doc="Path to JS renderer.",
     )
```

Comment:
using properties here will look better cc @friedkeenan

Reply:
If we do decide that's better it should wait for a later PR; the other changes here are much more important and this is already done with the other attributes.